docs/src/main/asciidoc/security-overview.adoc

```diff
@@ -73,10 +73,9 @@ For more information, see the xref:http-reference.adoc#same-site-cookie[SameSite
 
 [[secrets-engines]]
 === Secrets engines
-Secrets engines are components that store, generate, or encrypt data.
+You can use secrets engines with Quarkus to store, generate, or encrypt data.
 
-Quarkus provides comprehensive HashiCorp Vault support.
-For more information, see the link:{vault-guide}[Quarkus and HashiCorp Vault] documentation.
+Quarkus provides additional extensions in Quarkiverse for securely storing credentials, for example, link:{vault-guide}[Quarkus and HashiCorp Vault].
```

Comment:
Does Quarkus have extensions for/support any other secrets engines for storing, generating, or encrypting data?

Should we mention any of the other 'secrets engines' or is it just HashiCorp Vault that we recommend/support?

Thanks

Reply:
Good question @michelle-purcell, there is a file vault quarkiverse extension but we have Quarkus Configuration  system supporting it directly.
It would make sense to link to https://quarkus.io/guides/config-reference#secret-keys-expressions somehow (which has a follow up link to Smallrye Config)